### ISSUE TITLE
Support newer bytestring

### ISSUE THREAD

**Issue by kozross** [haskellfoundation/shields#2]
Currently blocked by [``snap-server``](https://github.com/snapframework/snap-server/issues/142).

**Comment by kozross** [comment]
No longer relevant as of dd9d1841844e0472af00da94efbf21a3e7ef2873, as we no longer depend on it directly.
